adoc/syclbase.adoc

```diff
@@ -64,6 +64,17 @@ include::config/attribs.adoc[]
 :cpp20: pass:[C++20]
 // Fortunately, the cpp macro for C++ is defined by default in AsciiDoctor
 
+// Use these to format a non-normative note.  The Asciidoc source:
+//
+//  {note}Caveat emptor.
+//  {endnote}
+//
+// is formatted like:
+//
+//  [Note: Caveat emptor. -- end note]
+:note: pass:q[[_Note:_ ]
+:endnote: pass:q,a[_&#8212;{nbsp}end{nbsp}note_]]
```

Comment:
Does `&mdash;` work by luck? Just a little more readable

Reply:
No, `&mdash;` causes the PDF converter to fail.  In fact, the [Asciidoc documentation](https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/) recommends using numeric Unicode values for this reason:

> However, using named character references can cause problems when generating non-HTML output such as PDF because the lookup table needed to resolve these names may not be defined. The recommendation is avoid using named character references, with the exception of the well-known ones defined in XML (i.e., lt, gt, amp, quot, apos). Instead, use numeric character references (e.g., \&#8364;).